website/www/site/content/en/documentation/dsls/dataframes/differences-from-pandas.md

```diff
@@ -51,20 +55,42 @@ Note that this collects the entire input dataset on a single node, so there’s
 
 ### Operations that produce non-deferred columns
 
+Examples:
+[`DeferredDataFrame.pivot`](https://beam.apache.org/releases/pydoc/current/apache_beam.dataframe.frames.html#apache_beam.dataframe.frames.DeferredDataFrame.pivot),
+[`DeferredDataFrame.transpose`](https://beam.apache.org/releases/pydoc/current/apache_beam.dataframe.frames.html#apache_beam.dataframe.frames.DeferredDataFrame.transpose),
+[`DeferredSeries.factorize`](https://beam.apache.org/releases/pydoc/current/apache_beam.dataframe.frames.html#apache_beam.dataframe.frames.DeferredSeries.factorize)
+
 Beam DataFrame operations are deferred, but the schemas of the resulting DataFrames are not, meaning that result columns must be computable without access to the data. Some DataFrame operations can’t support this usage, so they can’t be implemented. These operations raise a [WontImplementError](https://beam.apache.org/releases/pydoc/{{< param release_latest >}}/apache_beam.dataframe.frame_base.html#apache_beam.dataframe.frame_base.WontImplementError).
 
+<!-- TODO(BEAM-12169): Document the use of categorical columns as a workaround -->
```

Comment:
FYI @yeandy I added this TODO here (since I was editing this page anyway). This would be a good place to document how categorical columns can be used as a workaround for some non-deferred-column operations.